parser_test.go

```diff
@@ -7,12 +7,11 @@ import (
 )
 
 var (
-	timeout  = make(chan bool, 1)
 	gcChan   = make(chan *gctrace, 1)
 	scvgChan = make(chan *scvgtrace, 1)
 )
 
-func timeoutAfter(d time.Duration, t *testing.T) {
+func timeoutAfter(d time.Duration, timeout chan bool, t *testing.T) {
```

Comment:
Nah, your not stupid, your solution just evolved towards something that was
present in the standard library.
On 3 Mar 2015 08:08, "Vincent Bonmalais" <notifications@github.com> wrote:

> In parser_test.go
> <https://github.com/davecheney/gcvis/pull/12#discussion_r25638582>:
>
> >  	gcChan   = make(chan *gctrace, 1)
> >  	scvgChan = make(chan *scvgtrace, 1)
> >  )
> >
> > -func timeoutAfter(d time.Duration, t *testing.T) {
> > +func timeoutAfter(d time.Duration, timeout chan bool, t *testing.T) {
>
> Ahah I feel very stupid right now ;).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/davecheney/gcvis/pull/12/files#r25638582>.
>